contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -390,7 +439,7 @@ contract WithdrawalQueue {
      *
      * @return hint rate index for this request
      */
-    function findRateHint(uint256 _requestId) public view returns (uint256 hint) {
+    function findClaimingRateHint(uint256 _requestId) public view returns (uint256 hint) {
```

Comment:
maybe `findClaimRateHint` is a bit cleare

Reply:
Probably.